postinstall failed "Cannot find module nightwatch.conf.js"

I have a little web project with VueJS 2 and Webpack 2.

I encouter an issue when I want to add embed-js in my app :
```
$ yarn add embed-js
yarn add v0.21.3
[1/4] Resolving packages...
[2/4] Fetching packages...
warning fsevents@1.1.1: The platform "win32" is incompatible with this module.
info "fsevents@1.1.1" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
[4/4] Building fresh packages...
error C:\projects\myproject\node_modules\embed-js: Command failed.
Exit code: 1
Command: C:\windows\system32\cmd.exe
Arguments: /d /s /c node nightwatch.conf.js
Directory: C:\projects\myproject\node_modules\embed-js
Output:
module.js:471
    throw err;
    ^

Error: Cannot find module 'C:\projects\myproject\node_modules\embed-js\nightwatch.conf.js'
```

Any idea ?
Thanks in advance.

--- a/package.json
+++ b/package.json
@@ -85,7 +85,7 @@
     "rollup": "^0.41.4",
     "rollup-plugin-buble": "^0.15.0",
     "rollup-plugin-commonjs": "^7.0.0",
-    "rollup-plugin-filesize": "^1.0.1",
+    "rollup-plugin-filesize": "1.2",
     "rollup-plugin-node-resolve": "^2.0.0",
     "rollup-plugin-uglify": "^1.0.1",
     "saucelabs": "^1.4.0",

--- a/package.json
+++ b/package.json
@@ -102,9 +102,9 @@
     "build": "rollup -c rollup.umd.config.js && rollup.es2015.config.js",
     "semantic-release": "semantic-release pre && npm publish && semantic-release post",
     "prepublish": "grunt dist",
-    "e2e": "grunt dist && node_modules/.bin/nightwatch -e local",
-    "e2e:saucelabs": "grunt dist && node_modules/.bin/nightwatch -e chromemac,firefox",
-    "postinstall": "node nightwatch.conf.js"
+    "e2e": "grunt dist && npm run install:selenium && node_modules/.bin/nightwatch -e local",
+    "e2e:saucelabs": "grunt dist && npm run install:selenium && node_modules/.bin/nightwatch -e chromemac,firefox",
+    "install:selenium": "node nightwatch.conf.js"
   },
   "config": {
     "commitizen": {

